compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt

```diff
@@ -388,6 +388,7 @@ internal class ComposeAccessible(
                 Role.Checkbox -> AccessibleRole.CHECK_BOX
                 Role.RadioButton -> AccessibleRole.RADIO_BUTTON
                 Role.Tab -> AccessibleRole.PAGE_TAB
+                Role.DropdownList -> AccessibleRole.COMBO_BOX
```

Comment:
if the component has `Role.DropdownList` or `Role.Tab`, are there any additional actions like switch to tab, or press an item?

Reply:
"Switch to tab" is done by "clicking" the tab.
Pressing an item is done in the dropdown list itself. Note that `Role.DropdownList` is placed on the element/button that opens the dropdown list, not on the dropdown list itself.

For the dropdown list itself, perhaps we could translate `SemanticsProperties.SelectableGroup` to `AccessibleRole.LIST`, but that would be a different issue/PR.